test/acceptance.sh

```diff
@@ -183,4 +189,41 @@ for scenario in $(echo ${scenarios}); do
     done
 
   popd > /dev/null
-done
+done <<< "$(find "${__dir}/scenario" -type f -iname 'run.sh')"
+
+[[ "${1:-}" ]] && exit 0
+
+# finally do some shellcheck linting
+if [[ "$(command -v shellcheck)" ]]; then
+  echo "==> Shellcheck"
+  pushd "${__root}" > /dev/null
+
+  failed="false"
+
+  while IFS=$'\n' read -r file; do
+    lint="false"
+    [[ "$file" = "./main.sh" ]] && lint="true"
+    [[ "$file" = "./example.sh" ]] && lint="true"
+    [[ "$file" = "./test/acceptance.sh" ]] && lint="true"
+    [[ "$lint" != "true" ]] && continue
+
+    echo -n "    ${file}.. "
+
+    if ! shellcheck --shell=bash --external-sources "$file" >> "${__accptstTmpDir}/shellcheck.err"; then
```

Comment:
you're right. will fix.
would be cool to our own set of rules in shellcheck too.
don't know if this is possible. will research.